docs/setup/cumulus-aws-template.yaml

```diff
@@ -198,8 +198,6 @@ Resources:
               - !Sub "s3://${S3Bucket}/${EtlSubdir}/servicerequest"
               - !Sub "s3://${S3Bucket}/${EtlSubdir}/covid_symptom__nlp_results"
               - !Sub "s3://${S3Bucket}/${EtlSubdir}/covid_symptom__nlp_results_term_exists"
-              - !Sub "s3://${S3Bucket}/${EtlSubdir}/etl__completion"
-              - !Sub "s3://${S3Bucket}/${EtlSubdir}/etl__completion_encounters"
```

Comment:
I just did this so that folks don't reference these tables yet until we're 100% ready - if a crawler tries to crawl a dir that doesn't exist, and then you query what the metadata for that table is (as Library does, to see what it can use), you get errors. :frowning:

Reply:
do you want to comment them out instead?

Reply:
If this were code for devs, I'd be down for that - but since this is like, public template to be copied, I just want to avoid any questions about what these commented lines might mean until I'm ready to announce how to use this.

I probably should not have even added these here, before I was ready to go public.